pyvista/core/pointset.py

```diff
@@ -743,6 +786,11 @@ def save(self, filename, binary=True):
         one system may not be readable on other systems.  Binary can be used
         only ".vtk" files
 
+        Examples
+        --------
+        >>> import pyvista
+        >>> beam = pyvista.UnstructuredGrid(pyvista.examples.hexbeamfile)
+        >>> _ = beam.save('/tmp/tmp.vtk')  # or "C:/Users/where/tmp.vtk"
```

Comment:
This definitely should go into it's own commit. If there's something wrong with the rest and someone feels the need to revert, this should not get reverted.

Reply:
While I agree that this is outside the scope of this PR, we're constraining ourselves to lots of PRs (and 30 minutes of build time plus reviews).  I'll remove it here and make another PR in the future.